README.md

```diff
@@ -85,6 +85,14 @@ Which lock file format to use, defaults to _json_.
 * _json_, lock file in JSON format, default filename is dependency-lock.json
 * _pom_, lock file in POM XML format, default filename is .dependeny-lock/pom.xml
 
+### dependencyIntegrityChecking
```

Comment:
Maybe just checkIntegrity or just "integrity" since it both adds the entries and verifies them.

Reply:
changed to `checkIntegrity`